app/scripts/components/common/layout-root/index.tsx

```diff
@@ -35,17 +43,58 @@ const PageBody = styled.div`
 `;
 
 function LayoutRoot(props: { children?: ReactNode }) {
+  const useConsentForm = checkEnvFlag(process.env.COOKIE_CONSENT_FORM);
+
+  const readCookie = (name) => {
+    const nameEQ = name + '=';
+    const attribute = document.cookie.split(';');
+    for (let i = 0; i < attribute.length; i++) {
+      let c = attribute[i];
+      while (c.charAt(0) == ' ') c = c.substring(1, c.length);
+      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length, c.length);
+    }
+    return null;
+  };
   const { children } = props;
+  let cookieContents;
+
+  function getCookie() {
+    if (document.cookie != '') {
+      const cookie = readCookie('CookieConsent');
+      if (cookie != null) {
+        cookieContents = JSON.parse(cookie);
+
+        cookieContents.answer && setGoogleTagManager();
+
+        return cookieContents;
+      }
+    }
+    cookieContents = 'NO COOKIE';
+  }
+
+  const cookieConsentContent = getCookieConsentFromVedaConfig();
+  const showForm = () => {
+    getCookie();
```

Comment:
☝️  so this function can use the return value from getCookie without going outside of the function scope. What do you think?

Reply:
100% agree!